packages/amesos2/src/Amesos2_Superludist_FunctionMap.hpp

```diff
@@ -372,10 +372,14 @@ namespace Amesos2 {
     static void LUstructInit(SLUD::int_t m, SLUD::int_t n,
 			     type_map::LUstruct_t* lu)
     {
+#if defined(AMESOS2_ENABLE_SUPERLUDIST_5_4_0)
```

Comment:
This test may not be needed. The CMake 2 arg test will pass and the line number 378 will evaluate to true, right ?

Reply:
Unfortunately, it sometimes does not work. As we decide to support superlu dist version 5 and greater, the 2arg test won't be necessary.